[2024-06-13T09:35:59.697+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-13T09:35:59.713+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_prediction.test_model_task manual__2024-06-13T09:34:41.290979+00:00 [queued]>
[2024-06-13T09:35:59.719+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_prediction.test_model_task manual__2024-06-13T09:34:41.290979+00:00 [queued]>
[2024-06-13T09:35:59.720+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-13T09:35:59.729+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_model_task> on 2024-06-13 09:34:41.290979+00:00
[2024-06-13T09:35:59.737+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6140) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-13T09:35:59.739+0000] {standard_task_runner.py:63} INFO - Started process 6158 to run task
[2024-06-13T09:35:59.739+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stock_prediction', 'test_model_task', 'manual__2024-06-13T09:34:41.290979+00:00', '--job-id', '391', '--raw', '--subdir', 'DAGS_FOLDER/demo_***.py', '--cfg-path', '/tmp/tmpgxnhc1my']
[2024-06-13T09:35:59.740+0000] {standard_task_runner.py:91} INFO - Job 391: Subtask test_model_task
[2024-06-13T09:35:59.774+0000] {task_command.py:426} INFO - Running <TaskInstance: stock_prediction.test_model_task manual__2024-06-13T09:34:41.290979+00:00 [running]> on host 33b4ab31339c
[2024-06-13T09:35:59.844+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nguyen Khoa Doan' AIRFLOW_CTX_DAG_ID='stock_prediction' AIRFLOW_CTX_TASK_ID='test_model_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-13T09:34:41.290979+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-13T09:34:41.290979+00:00'
[2024-06-13T09:35:59.846+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-13T09:36:00.547+0000] {logging_mixin.py:188} INFO -  1/17 ━━━━━━━━━━━━━━━━━━━━ 3s 224ms/step
[2024-06-13T09:36:00.597+0000] {logging_mixin.py:188} INFO - 14/17 ━━━━━━━━━━━━━━━━━━━━ 0s 4ms/step  
[2024-06-13T09:36:00.763+0000] {logging_mixin.py:188} INFO - 17/17 ━━━━━━━━━━━━━━━━━━━━ 0s 13ms/step
[2024-06-13T09:36:00.764+0000] {logging_mixin.py:188} INFO - 17/17 ━━━━━━━━━━━━━━━━━━━━ 0s 14ms/step
[2024-06-13T09:36:00.801+0000] {warnings.py:112} WARNING - /opt/***/dags/demo_***.py:291: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  test_data1['Dự đoán'] = y_test_predict  # thêm dữ liệu

[2024-06-13T09:36:01.362+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-13T09:36:01.363+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-13T09:36:01.373+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=stock_prediction, task_id=test_model_task, run_id=manual__2024-06-13T09:34:41.290979+00:00, execution_date=20240613T093441, start_date=20240613T093559, end_date=20240613T093601
[2024-06-13T09:36:01.440+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-13T09:36:01.455+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-13T09:36:01.457+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
